core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3234,6 +3234,41 @@ paths:
             'application/json':
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/fileTransfers:
+    get:
+      tags:
+        - workspaces
+      summary: List pending file transfers for this workspace
+      description: List pending file transfers for this workspace
+      operationId: listPendingFileTransfers
+      parameters:
+        - name: workspaceNamespace
+          in: path
+          description: Workspace Namespace
+          required: true
+          schema:
+            type: string
+        - name: workspaceName
+          in: path
+          description: Workspace Name
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Successful Request
+          content:
+            'application/json':
+              schema:
+                type: array
```

Comment:
Note that this endpoint returns a list of `PendingCloneWorkspaceFileTransfer`s. 

In reality, it will currently only ever return 0 or 1 `PendingCloneWorkspaceFileTransfer`s. I personally find the list approach to be easier to consume. The other benefit here is that this is future-proofed for if/when we ever add file transfers as a general feature.

The alternative approach would be to return 204 in the case of none, 200 + JSON in the case of a pending transfer, and 404 otherwise.

Reply:
Returning a list sounds good to me. The future proofing argument makes sense and checking the list's length seems cleaner than dealing with multiple different status codes.

Reply:
I think this makes sense, and I like the future proofing.